src/planscape/testing/postgis_setup.sql

```diff
@@ -0,0 +1,8 @@
+CREATE EXTENSION postgis;
+CREATE EXTENSION postgis_raster;
+ALTER SYSTEM SET postgis.enable_outdb_rasters TO True;
```

Comment:
why are we using outdb rasters here?

Reply:
This is necessary to convert any raster into an external format, e.g., PNG or JPEG for displaying tiles.

Reply:
Ack, thanks